src/services/floating-manager/floating-manager.tsx

```diff
@@ -201,5 +202,7 @@ export class FloatingManager {
       this._updateCachedCanvas();
       this._updateComponents();
     });
+
+    this._eventManager.listen(kalturaPlayer, ToastEvent.SHOW_TOAST, () => this._updateComponents());
```

Comment:
you can do something like
UiManagerEvent.UPDATE_COMPONENTS
and then its more generic and we'll be able to fire it from other places afterwards without needing to refactor this file